release.config.js

```diff
@@ -0,0 +1,59 @@
+// Release steps
+
+const verifyConditions = [
+  [
+    '@semantic-release/git',
+    {
+      assets: ['package.json'],
+    },
+  ],
+];
+
+const analyzeCommits = [
+  [
+    '@semantic-release/commit-analyzer',
+    {
+      preset: 'angular',
+      releaseRules: [
+        { type: 'docs', release: 'patch' },
+        { type: 'style', release: 'patch' },
+        { type: 'refactor', release: 'patch' },
+        { type: 'chore', release: 'minor' },
+        { scope: 'no-release', release: false },
+      ],
+    },
+  ],
+];
+
+const generateNotes = ['@semantic-release/release-notes-generator'];
+
+const prepare = [
+  ['@semantic-release/npm'],
+  [
+    '@semantic-release/git',
+    {
+      assets: ['package.json'],
+      message: 'chore(release): bump version ${nextRelease.version}',
+    },
+  ],
+];
+
+const publish = [['@semantic-release/npm'], ['@semantic-release/github']];
+
+// TODO configure following release steps, at the moment skipped
+const verifyRelease = [];
+const fail = [];
+const success = [];
+
+module.exports = {
```

Comment:
Die Plugins müssen doch in den Key "plugins" hinzugefügt werden. Ansonsten werden die Standard-Plugins (inkl. Standard-Konfiguration) geladen. Oder sehe ich das falsch?
https://semantic-release.gitbook.io/semantic-release/usage/plugins#plugins-declaration-and-execution-order